Frends.RabbitMQ.Publish/Frends.RabbitMQ.Publish.Tests/Lib/Helper.cs

```diff
@@ -24,10 +25,10 @@ internal static void ReadMessage(ReadValues readValues, Connection connection)
                 break;
         }
 
-        IConnection _connection = factory.CreateConnection();
-        IModel _model = _connection.CreateModel();
+        IConnection _connection = await factory.CreateConnectionAsync();
+        IChannel _model = await _connection.CreateChannelAsync();

```

Comment:
As per code rabbit comment above - can you check if we need `using` here as well?